lib/api/swagger.js

```diff
@@ -19,6 +19,8 @@
  * limitations under the License.
  */
 
+/* eslint sort-keys: 0 */
```

Comment:
Is this valid only for this file or until the rule is re-activated?

Reply:
If the rule is in the outer scope, it works for the entire file. BUT, I should have used _disable_ instead, as suggested in the [docs nitpicky](https://eslint.org/docs/user-guide/configuring.html#disabling-rules-with-inline-comments)

```suggestion
/* eslint-disable sort-keys */
```